src/Documentation/Advanced-Concepts/Cancellation-Tokens.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Grain cancellation tokens
+---
+
+# Grain cancellation tokens
+
+The Orleans runtime provides mechanism called grain cancellation token, that enable the developer to cancel asynchronous or long-running synchronous operations. 
```

Comment:
I think if instead of "to cancel asynchronous or long-running synchronous operations" we say "to cancel an executing grain operation", it would be more clear.
Also a typo: "enable" -> "enables".